### PR TITLE
TemperatureMatrix property for AMD88xx sensor

### PR DESCRIPTION
**Pull request by oliverchristen**
This PR implements a property to get the temperature image as a two-dimensional array.

### CODE REVIEW

**Review by pgrawehr — APPROVED** [review]
LGTM

File: src/devices/Amg88xx/Amg88xx.cs [hunk 1]
```diff
@@ -95,6 +95,29 @@ public Amg88xx(I2cDevice i2cDevice)
...
+        /// <summary>
+        /// Gets temperature for all pixels from the current thermal image as a two-dimensional array.
+        /// First index specifies the x-coordinate of the pixel and second index specifies y-coordinate of the pixel.
+        /// </summary>
+        /// <returns>Temperature as a two-dimensional array.</returns>
+        public Temperature[,] TemperatureMatrix
+        {
+            get
+            {
+                Span<byte> buffer = _imageData;
```
Comment (Ellerbach) [inline src/devices/Amg88xx/Amg88xx.cs:107]: This doesn't seem to be used?

**Review by Ellerbach — COMMENTED** [review]
Looks good to me. Please check the _imageData and buffer, not sure it should be there

File: src/devices/Amg88xx/Amg88xx.cs (hunk 1 above)
Comment (oliverchristen) [reply src/devices/Amg88xx/Amg88xx.cs]: Absolutely right. Leftover of a refactoring.

**Comment by fehdem** [comment]
@oliverchristen You may consider naming it ```TemperatureImage``` from a semantical perspective. This would be a better match to the other methods and documentation. Please add this also to the API description in the README.MD.

**Comment by oliverchristen** [comment]
@fehdem Agree. I renamed the property and added it to the API description.

**Comment by oliverchristen** [comment]
@fehdem I think build pipelines didn't fail because of my changes, did they?

**Comment by fehdem** [comment]
@oliverchristen No, I don't think so. The error message points to a problem somewhere else.

**Comment by oliverchristen** [comment]
@fehdem how do we proceed here with the failing tests? I think it's not related to the PR.

**Comment by pgrawehr** [comment]
@oliverchristen Try a rebase, that will at least restart the build

**Comment by pgrawehr** [comment]
@oliverchristen That's good now (two of the builds often fail, that's due to some hardware issues in the test lab). However, you should have rebased, not merged, so that only your commits stay in the PR.

**Comment by oliverchristen** [comment]
> @oliverchristen That's good now (two of the builds often fail, that's due to some hardware issues in the test lab). However, you should have rebased, not merged, so that only your commits stay in the PR.

Weird, I did rebase it. But somehow the rebased commits came into this PR anyway.

**Review by krwq — APPROVED** [review]

**Comment by krwq** [comment]
@Ellerbach do you want to also review this again?

**Pull request merged**